kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -41,6 +43,20 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
 
   @volatile private var shutdown = false
 
+  protected var _operationLogRoot: String = null
+
+  def operationLogRoot: String = _operationLogRoot
+
+  private def initOperationLogRootDir(): Unit = {
```

Comment:
We need init it in session manager, but I am not sure whether throw the exception if it failes

Reply:
maybe we can make operationLogRoot and Option[String], if we fail to initialize the _operationLogRoot, then we don't initialize the OperationLog for operations and just return the result

Reply:
or maybe we just remove this step for now

Reply:
i think it should throw the exception or logger it in the log.

when user set this configuration item by themself, this dir perhaps has no permission for KYUUBI to create. It should let user to know.

Reply:
> i think it should throw the exception or logger it in the log.
> 
> when user set this configuration item by themself, this dir perhaps has no permission for KYUUBI to create. It should let user to know.

It seems that you misunderstand something here, the config is from sessionManager not from users

Reply:
user set this configuration item.
KYUUBI create this dir.